app/components/input/input.css

```diff
@@ -1,5 +1,6 @@
 .Input {
```

Comment:
The changes to `input.css` and `icon.css` are actually unrelated to this #157, but a follow-up to #169